resources/public/contracts/src/OfferingFactory.sol

```diff
@@ -35,7 +35,7 @@ contract OfferingFactory {
     * @dev Registers new offering to OfferingRegistry, clears offering requests for this ENS node
     * Must check if creator of offering is actual owner of ENS name and for top level names also registration owner
     * @param node bytes32 ENS node
-    * @param labelHash bytes32 ENS labelhash
+    * @param labelHash bytes32 ENS label hash
```

Comment:
This is [specific ENS terminology](https://docs.ens.domains/contract-api-reference/name-processing#how-do-i-find-the-labelhash-namehash-of-a-name), I'd prefer to keep it.

Reply:
Oh, I thought it's `label hash` (separated). Good to know.